frontend/packages/ceph-storage-plugin/src/components/block-pool/dashbaord/mirroring-card-body.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import './mirroring-card.scss';
+
+export const MirroringCardBody: React.FC<MirroringCardBodyProps> = ({ children }) => (
+  <dl className="co-dashboard-card__body--top-margin ceph-block-pool-mirroring-card__body co-dashboard-text--small">
```

Comment:
Copied from <DetailsBody>.  <DetailItem> is always wanted title. But in mirroring cards sometimes i need a title and sometimes not.

And I also referred to existing cluster dashboard cards, every card has its own JSX.Element, so i created one for mirroring but almost similar to details card JSX.Element